ui/voice/src/main.ts

```diff
@@ -127,31 +131,6 @@ function voiceSettings(ctrl: VoiceMoveCtrl): VNode {
   ]);
 }
 
-let lastPointerDown = -1;
-document.addEventListener('pointerdown', e => (lastPointerDown = e.timeStamp));
-
-function toggleSettings(onButton: boolean) {
-  const settingsButton = $('#voice-settings-button');
-  const settingsPane = $('#voice-settings');
-
-  function onPointerUp(e: PointerEvent) {
-    const el = e.target as Element;
-    if (settingsPane.get(0)?.contains(el) || settingsButton.get(0)?.contains(el) || e.timeStamp - lastPointerDown > 500)
-      return;
-    toggleSettings(false);
-    document.removeEventListener('pointerup', onPointerUp);
-  }
-  // TODO - keyboard nav, nvui, use blur, something something
-  document.removeEventListener('pointerup', onPointerUp);
-
-  if (onButton || settingsButton.hasClass('active')) settingsButton.toggleClass('active');
-  if (settingsButton.hasClass('active')) {
-    settingsPane.show();
-
-    setTimeout(() => document.addEventListener('pointerup', onPointerUp));
-  } else settingsPane.hide();
-}
-
```

Comment:
should be addressed in https://github.com/lichess-org/lila/pull/12652/commits/15f79e9bf6827fc60b04628c024f7d12168536ec

Reply:
yes, it's so that mobile users can drag scroll without hiding the open menu.  it usually shows up under the chessboard on mobile so after disclosing they have to scroll down to see the contents.